### DISCUSSION TITLE
How do you make the note style change mid song?

### DISCUSSION THREAD

**Discussion by NK-523** [KadeArchive/Kade-Engine#2189] [Q&A]
How do you make the note style change mid song? Like in the Bob and Bosip mod or the Sunday mod. (And VS Eteled technically)

**repost of https://github.com/KadeDev/Kade-Engine/discussions/2114 because no one answered**

**Comment by BoloVEVO** [comment]
I saw your question days ago. I tried through modchart and playstate.hx, note.hx but nothing worked cuz my programming skills are trash. Look for a mod that changing note style mid song and use its source code.

**Comment by NK-523** [comment]
Found the code I need.
![image](https://user-images.githubusercontent.com/77185327/133112021-e257d164-8604-452d-a089-67c536962ed5.png)
but where to put it is the question.

**Reply by ghost** [reply]
At line 1853 i think
lol

**Reply by NK-523** [reply]
If only it were that easy.
